.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -9,6 +9,19 @@ body:
         Found a bug? Please fill out the sections below. 👍
         Thanks for taking the time to fill out this bug report!
         For backend issues, use [lemmy](https://github.com/LemmyNet/lemmy)
+  - type: checkboxes
+    attributes:
+      label: Requirements
+      description: Before you create a bug report please do the following.
+      options:
+        - label: Did you check to see if this issue already exists?
+          required: true
+        - label: Is this only a single bug. Do not put multiple bugs in one issue.
+          required: true
+        - label: Is this a question or discussion? Don't use this, use https://lemmy.ml/c/lemmy_support
```

Comment:
The logic is reversed here compared to the first two checks. For me now it reads as if I had to confirm that _"this is a question or discussion"_.

I think we should either formulate it the same way as the first check, like "Is this a bug report or feature request? For questions or discussions, use ..."

Or we could get rid of the question style all along:
* This issue does not already exists.
* This contains only a single bug. Do not ...
* This is a bug report or a feature request. For ...
* This is a client-side issue (related to the UI in the browser). For backend issues ...

Reply:
I agree and thought about it a bit but decided not to change it too much to honor the spirit of just adding the boxes. However, I'll take the liberty to rework is to be like your first suggestion (rather than change them all).

Thanks :)